tests/unit/test_instance_cmds.py

```diff
@@ -2939,4 +2952,4 @@ def test_execute_pywbemcli(self, desc, inputs, exp_response, mock,
         Execute pybemcli with the defined input and test output.
         """
         self.command_test(desc, self.command_group, inputs, exp_response,
-                          mock, condition, verbose=False)
+                          mock, condition)
```

Comment:
In removing the verbose=False, we are sort of hiding the fact that this exists.  That makes sense of general code but I think it would be easier for testers if the default was in place as a reminder.  This was done for all tests.

Reply:
That is the case for any parameter that the caller does not specify. Are you arguing that a caller should never benefit from parameter defaults?

Reply:
NO, just noting that I often use that variable and it is easy to remember if it is  there whether True or False.  Old peoples memory thing.

Reply:
This is only because this is in test code and having the variable there is a visual reminder when you are trying to figure out why a test blew up.